ogticrd/covid-safe-paths
# changing language in prod mode

#### Description:


Adding the spanish language in production mode, before, when you create an APK just showed you english and other rare language, but now it shows english and spanish
#### Linked issues:


Fixe [#114](https://trello.com/c/z9VU5Fvl/114-fix-languages-in-prod)
#### Screenshots:



#### How to test:


- `git pull`
- `git checkout Fix/114-languages-in-production`
- create an APK with this branch and in settings or in the beginning of the app check the languages
- it should show you english and spanish only
